packages/plugin-cloudflare/scripts/postinstall.js

```diff
@@ -52,6 +51,8 @@ function getBinPathTarget() {
 }
 
 export default async function install() {
+  // Don't install cloudflare if the SHOPIFY_CLI_IGNORE_CLOUDFLARED environment variable is set
+  if (process.env.SHOPIFY_CLI_IGNORE_CLOUDFLARED) return
```

Comment:
this is added because of some very specific issue with a partner, we don't have the other flag (`SHOPIFY_CLI_CLOUDFLARED_PATH`) documented either.

I think we don't want people to actively ignore this no?

Reply:
Yeah, it will only be useful for orgs which don't allow tunnels I guess. Anyway, I think It would be nice to have all the flags documented somewhere, but no need to do it now.